src/Robo/Plugin/Commands/SiteCommands.php

```diff
@@ -138,14 +138,15 @@ public function siteInstall() {
   public function siteUpdate() {
     $this->validateConfig();
     $drush = $this->drushExecutable();
-    $execStack = $this->taskExecStack()->stopOnFail(TRUE);
     $commands = [];
 
     if ($this->isDrush9()) {
       $commands[] = "state-set system.maintenance_mode TRUE";
 
       // Allow updatedb to fail once and execute it again after config:import.
-      $commands[] = "updatedb -y";
+      $this->taskExec("{$drush} updatedb -y")->run();
```

Comment:
La fel trebuie facut si pentru Drupal 7 si pentru Drupal 8. Deci ar trebui sa muti linia asta si urmatoare in afara if-ului (definitia lui $execStack era un trebuie, mai trebuia doar sa adaugi linia asta inainte)